roles/letsencrypt/meta/main.yml

```diff
@@ -0,0 +1,21 @@
+dependencies:
+  - role: apache
+    ssl: no
+
+    apache_mods_enabled:
+      - rewrite.load
+
+    apache_vhosts:
```

Comment:
We have to change all our web hook URLs anyway because we're moving to an Integration, where we can control the web hook URL instead of asking customers to do it.